src/utils/trust.former.bo.ts

```diff
@@ -40,7 +42,7 @@ const getPageProperties = (
   formData?: PageModel.TrustHistoricalBeneficialOwnerForm,
   errors?: FormattedValidationErrors,
 ): TrustHistoricalBeneficialOwnerProperties => ({
-  backLinkUrl: getTrustInvolvedUrl(isUpdate, trustId),
+  backLinkUrl: getTrustInvolvedUrl(isUpdate, trustId, req),
```

Comment:
can you add a check into the tests to check the back link is the 'with params' version. We've modified the getTrustInvolvedUrl to use the feature flag and this is also used for the back link on line 45.  I missed it in my ticket and have put up a new PR to cover it for the trust BO individual.

Reply:
Added check.